deploy/crds/compliance.openshift.io_v1alpha1_complianceremediationcheck_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceCheckResult
+metadata:
+  name: example-compliancecheckresult
+spec:
+  # Add fields here
+  size: 3
```

Comment:
Let's either add a real example here. Or just remove this file.

Reply:
fixed locally